cg/models/orders/samples.py

```diff
@@ -265,20 +265,9 @@ class MicrobialSample(OrderInSample):
     well_position: str | None
     # "Required if "Other" is chosen in column "Species""
     organism_other: constr(max_length=Organism.internal_id.property.columns[0].type.length) | None
-    # "These fields are not required"
-    concentration_sample: float | None
-    quantity: int | None
```

Comment:
This will affect covid orders as well - is that planned?

Reply:
No it will not. We tested it with a sars order too